integration_tests/test_stage.py

```diff
@@ -46,3 +46,32 @@ def test_conflicts(self):
             '`snapcraft help plugins`'
         )
         self.assertThat(exception.output, Contains(expected_help))
+
+    def test_no_conflicts(self):
+        project_dir = 'no-conflicts'
+        self.run_snapcraft('stage', project_dir)
+
+    def test_post_organize_conflicts(self):
+        project_dir = 'post-organize-conflicts'
+
+        exception = self.assertRaises(
+            subprocess.CalledProcessError,
+            self.run_snapcraft, 'stage', project_dir)
```

Comment:
You're only using `stage` here, which means you can operate on individual parts. Since in the snaps you created the code being build is identical, I suggest you combine them into a single snap with multiple parts, two of which don't conflict and two of which do conflict. Otherwise we end up with two identical snaps except for a single organize key. Would that work?